page-factory-core/src/main/java/ru/sbtqa/tag/stepdefs/CoreSetupSteps.java

```diff
@@ -1,5 +1,6 @@
 package ru.sbtqa.tag.stepdefs;
 
+import com.sun.jmx.snmp.tasks.Task;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.sun.jmx.snmp.tasks.Task'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOE5crQ7oSxgxngC&open=AWeIHOE5crQ7oSxgxngC)

[ISSUE_KEY]: AWeIHOE5crQ7oSxgxngC